open-sphere-plugins/extra-icons/src/main/java/io/opensphere/milstdi/mantle/controller/StandardIcons.java

```diff
@@ -0,0 +1,217 @@
+package io.opensphere.milstdi.mantle.controller;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import org.apache.log4j.Logger;
+
+import gnu.trove.list.TIntList;
+import gnu.trove.list.array.TIntArrayList;
+import io.opensphere.core.util.collections.New;
+import io.opensphere.core.util.lang.StringUtilities;
+import io.opensphere.mantle.MantleToolbox;
+import io.opensphere.mantle.icon.IconProvider;
+import io.opensphere.mantle.icon.IconRecord;
+import io.opensphere.mantle.icon.impl.IconProviderFactory;
+
+/** Way to access MilStd-2525c icons. */
+public final class StandardIcons
+{
+    /** Logger. */
+    private static final Logger LOGGER = Logger.getLogger(StandardIcons.class);
+
+    /** The images file path. */
+    private static final String IMAGES_FILE_PATH = "/images/standard/milstd2525c.txt";
+
+    /** Map for storage of default Google Earth icons. */
+    private static Map<String, URL> ourIconHrefToUrlMap;
+
+    /**
+     * Gets the icon map.
+     *
+     * @param mantleToolbox Reference to the MantleToolbox
+     * @return the icon map
+     */
+    public static synchronized Map<String, URL> getIconMap(MantleToolbox mantleToolbox)
```

Comment:
This class has a lot in common with another one above, should they be refactored to share some code?  Up to you, just something I noticed.